src/main/kotlin/Composite/base/Kitchen.kt

```diff
@@ -0,0 +1,13 @@
+package Composite.base
+
+class Kitchen(private var cookers: MutableList<Cooker> = mutableListOf()) : Cooker {
+  override fun cook() {
+    cookers.forEach {it.cook()}
+  }
+
+  fun add(cooker: Cooker) = cookers.add(cooker)
+
+  fun remove(cooker: Cooker) {
+    cookers = cookers.filterNot { it == cooker } as MutableList<Cooker>
```

Comment:
Omg, I was complicating myself too much 😨 

You can change it yourself and learn how to collaborate on an PR 😉